src/Console/Report/FixReport/TextReporter.php

```diff
@@ -41,6 +43,7 @@ public function generate(ReportSummary $reportSummary): string
                 $output .= $this->getAppliedFixers(
                     $reportSummary->isDecoratedOutput(),
                     $fixResult['appliedFixers'],
+                    $fixResult['extraInfoFixers']
```

Comment:
I really wonder if we need to change the `fixResult` object, or simply have one standalone method that is converting `fixer-name` to `fixer-url`

Reply:
I will let you think again then ...